tests/quizzes/good/image-questions/quiz.json

```diff
@@ -0,0 +1,35 @@
+{
+      "title": "Image Quiz",
```

Comment:
This title does not match the dir name.
The difference in spacing and casing is fine, but then this should be "Image Questions".